requirements/app.in

```diff
@@ -1,5 +1,4 @@
-flexmeasures>=0.9.3
```

Comment:
I guess there might have been a reason for 0.9.3 instead of 0.9

Let's keep 0.9.3

Reply:
Oh yes, thanks.